File: app/src/main/java/com/example/todonotes/MainActivity.java
package com.example.todonotes;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.util.Log;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Intent intent = getIntent();
        intent.getStringExtra("full_name");
        // intent.getFloatExtra("float_data");
        Log.d("myNotesActivity", intent.getStringExtra("full_name"));

    }
}

File: app/src/main/java/com/example/todonotes/MainActivity.kt
package com.example.todonotes

import android.os.Bundle
import android.util.Log
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val intent = intent
        intent.getStringExtra("full_name")
//         intent.getFloatExtra("float_data");
        Log.d("myNotesActivity", intent.getStringExtra("full_name")!!)
    }
}
